docs/faq.md

```diff
@@ -4,7 +4,16 @@
 
 If you have USB debugging enabled in developer options, connect your phone to the PC. If your device is detected (check by `adb devices`), enter ADB shell and run the command `magisk --remove-modules`. This will remove all your modules and automatically reboot the device.
 
-If unfortunately you do not have USB debugging enabled, reboot into Safe Mode. Most modern Android devices support pressing a special key combo at boot to enter Safe Mode as an emergency option. Magisk will detect Safe Mode being activated, and all modules will be disabled. Then reboot back to normal mode (the module disable state persists) and manage your modules through the Magisk app.
+If unfortunately you do not have USB debugging enabled, reboot into Safe Mode. Most modern Android devices support a special key combo at boot to enter Safe Mode as an emergency option and Magisk can detect the Safe Mode key combo in a similar way to system to disable Magisk modules.
+
+Please Note: Magisk detects the key combo earlier than system, so it's possible to activate Magisk Safe Mode but not system Safe Mode and vice versa. Key combo detection can therefore be inconsistent but the following details should ensure that modules are properly disabled:
+1) Many online guides for entering Safe Mode say 'When the animated logo appears, press and hold the volume down button until the system boots' or similar. This may actually be too late for Magisk detection however and result in activating system Safe Mode but modules aren't disabled.
+2) By pressing the volume down button some seconds before the animation and releasing it when the boot animation appears, Magisk Safe Mode should be activated without activating system Safe Mode and the device should then simply boot to normal system with modules disabled.
+3) By pressing the volume down button some seconds before the animation and holding it until the system boots, both Magisk Safe Mode and system Safe Mode should be activated. Next, after rebooting back to normal system modules will be disabled.
+
+Varying the key combo timing that online guides give for entering system Safe Mode is often needed to activate Magisk 'Safe Mode' as described above, but this will add an empty directory named 'disable' to the modules directory to disable modules when next booted with Magisk thereby bypassing module related bootloop conditions.
```

Comment:
Slight tweaks I'd recommended for accuracy and clarity:

*but in any case this

*empty file

Reply:
Thanks... Maybe not 'in any case' since point is it'll only happen in second case (ie. when timing is varied) which is the point of this clarification. 🙃... Will try to improve it though.

... Ah yes, empty file...

Haven't discovered how to add bolding, italics etc in these edits strangely... Please say what I'm blind to! 😬

Reply:
Hope that's better... Thanks for input/reviewing! 👍